config/initializers/color_themes.rb

```diff
@@ -0,0 +1,18 @@
+require 'pathname'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.